gradle/versions.gradle

```diff
@@ -20,5 +20,5 @@ ext.versions = [
 	'gson_orbit': '2.8.9.v20220111-1409',
 	'websocket_jakarta': '2.0.0',

```

Comment:
should we bump xtend_lib and websocket_jakarta versions too?

Reply:
:+1: - the websocket jakarta needs a newer version in Orbit though for that to work???